triton/apps/importantnotes/resetconda.rst

```diff
@@ -0,0 +1,32 @@
+.. admonition:: Resetting conda
+   :class: toggle
+
+   Sometimes it is necessary to reset your Conda configuration. So here are instructions on how to wipe all
+   of your conda settings and existing environments. To be able to do so first load the miniconda environment 
+
+   .. code-block:: bash
+   
+      module load miniconda
+
+   First, check where conda stores your environments:
+
+   .. code-block:: bash
+   
+     conda config --show env_dirs     
+     conda config --show pkgs_dirs
+     
+   Delete the directories that are listed and start with ``/home/<username>`` (this could e.g. be ``/home/<username>/.conda/envs``)
+   and ``/scratch/`` ( e.g. ``/scratch/work/<userame>/conda_envs``). 
```

Comment:
```suggestion
   Delete the directories that are listed and start with ``/home/USERNAME`` (this could e.g. be ``/home/<username>/.conda/envs``)
   and ``/scratch/`` ( e.g. ``/scratch/work/USERNAME/conda_envs``). You would delete
   these with ``rm -r DIRNAME``, but be careful you use the right paths because there
   is no going back.
```